scripts/litePageSizeValidator/index.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
+const { exec } = require('child_process');
```

Comment:
Can we use import instead of require?
```suggestion
import { exec } from 'child_process';
```

Reply:
we can but then you can't run it locally without adding type: module into the package.json

Reply:
OK, let's leave it for now., thanks